azavea/raster-vision
# Update docs and version number

## Overview

This PR updates the docs and updates all occurrences of the version number from `0.12` to `0.13`.

Depends on #1116 

### Checklist


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as rebuilding the Docker image.
* Include test case, and expected output if not captured by automated tests.

Makes progress towards #1085
